web/templates/settings/admin_user_list.hbs

```diff
@@ -1,6 +1,6 @@
 <tr class="user_row{{#if is_active}} active-user{{else}} deactivated_user{{/if}}" data-user-id="{{user_id}}">
     <td class="user_name panel_user_list">
-        {{> ../user_display_only_pill display_value=full_name user_id=user_id img_src=img_src}}
+        {{> ../user_display_only_pill display_value=full_name user_id=user_id img_src=img_src is_active=is_active}}
```

Comment:
@andersk hmm, I think this template takes considerably more parameters, and at least one caller passes the following parameters:

```
function human_info(person: User): {                                                                    
    is_bot: false;                                                                                      
    user_role_text: string | undefined;                                                                 
    is_active: boolean;                                                                                 
    user_id: number;                                                                                    
    full_name: string;                                                                                  
    bot_owner_id: number | null;                                                                        
    can_modify: boolean;                                                                                
    is_current_user: boolean;                                                                           
    cannot_deactivate: boolean;                                                                         
    display_email: string | null;                                                                       
    img_src: string;                                                                                    
    last_active_date: string;                                                                           
} {                                                                                                     
```

`web/templates/user_display_only_pill.hbs`

Reply:
I'm not sure which of those extra parameters were intended to be processed by this template; maybe just `is_current_user`?

Reply:
Added `is_current_user` and determined the rest are not necessary for this call point. Also `admin_user_list.hbs` has a confusing template name, fixed in 3d5dbefcb527e44ab471d49946e8e34f0fbed482.